pelita/datamodel.py

```diff
@@ -115,6 +115,13 @@ def is_destroyer(self):
         return self.in_own_zone
 
     @property
+    def on_left_side(self):
+        # True if current_pos is left of team_border
+        return (self.current_pos[0] < ((sorted(self.homezone)[1] + 1)
```

Comment:
`sorted(self.homezone)[1]` is equivalent to `max(self.homezone)`, right?

Reply:
And that nested if is a little convoluted. I think a verbose multi-line if is better here.

Reply:
@Debilski, I thought I would have to import math or numpy for a max function ... just realized now that it's not necessary to import anything for that :-) 

Next commit will be more verbose